src/components/textInput/TextInputBase.tsx

```diff
@@ -464,6 +468,7 @@ export const TextInputBase = ({
           selectionColor={IOColors[theme["interactiveElem-default"]]} // Caret on iOS
           cursorColor={IOColors[theme["interactiveElem-default"]]} // Caret Android
           maxLength={adjustedMaxLength}
+          maxFontSizeMultiplier={IOMaxFontSizeMultiplier}
```

Comment:
What do you think about allowing it to be passed from the outside as well, instead of relying solely on a default?

Reply:
@shadowsheep1 Unless we need to set a specific value, I'd leave it as it is. The `IOMaxFontSizeMultiplier` value was set based on an analysis of user preferences. For example, `1.2` is too low for some of our citizens.

While I understand that it could be useful in some cases, if we start adding exceptions, the experience for people with low vision will become inconsistent.